index.html

```diff
@@ -53,34 +53,32 @@ <h3 class="shadow">Startujemy już wkrótce!</h3>
     </div>
 </header>
 <main>
-    <section id="product-icons-section" class="flex-container-center">
-        <!-- krótki opis funkcjonalności produktu -->
+    <section id="icons" class="flex-container-center section">
         <div class="flex-container-center">
-            <div class="icons-display-container">
+            <div class="icons-container">
                 <img class="icons-images" src="images/icons/car-mechanic.svg" alt="car icon">
-                <h2 class="icons-text">Tysiące części<br>dla każdego<br>samochodu</h2>
+                <h2 >Tysiące części<br>dla każdego<br>samochodu</h2>
```

Comment:
This one should be corrected. Do not leave empty spaces inside markup names.

Reply:
Fixed.